src/api/rpc_api/author.rs

```diff
@@ -106,13 +106,77 @@ pub trait SubmitAndWatch {
 
 	/// Submit an extrinsic and watch it until the desired status
 	/// is reached, if no error is encountered previously.
+	///
+	/// If watched until `InBlock` or `Finalized`, this function will
+	/// return an error if the extrinsic was not successfully executed.
+	/// If it was successful, a report containing the following is returned:
+	/// - extrinsic hash
+	/// - hash of the block the extrinsic was included in
+	/// - last known extrinsic (transaction) status
+	/// - associated events of the extrinsic (only for InBlock or Finalized)
```

Comment:
```suggestion
	/// - associated events of the extrinsic
```

Optional, since it's already mentioned above. If you'd rather keep it, you should also put it behind the bullet point `hash of the block the extrinsic was included in` for consistency. I'm fine with either.